src/main/java/net/atos/entng/calendar/controllers/CalendarController.java

```diff
@@ -149,7 +149,7 @@ public void deleteCalendar(HttpServerRequest request) {
                 .onFailure(err -> renderError(request));
     }
 
-    @Put("/:id/ical")
+    @Put("/:id/url")
```

Comment:
change route because it was the same as the ical import (will be fixed in later tickets)